docs/includes/ksql-includes.rst

```diff
@@ -300,15 +337,60 @@ the latest offset.
         CSAS_PAGEVIEWS_FEMALE_LIKE_89 | pageviews_enriched_r8_r9 | CREATE STREAM pageviews_female_like_89 WITH (kafka_topic='pageviews_enriched_r8_r9', value_format='DELIMITED') AS SELECT * FROM pageviews_female WHERE regionid LIKE '%_8' OR regionid LIKE '%_9';
         ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 
+#.  Optional: Examine query run-time metrics and details. Observe that information including 
+    the target Kafka topic is available, as well as throughput figures for the messages being processed.
+
+    .. code:: sql
+
+        ksql> DESCRIBE EXTENDED PAGEVIEWS_REGIONS;
+
+    Your output should resemble:
+
+    .. code:: bash
+
+        Name                 : PAGEVIEWS_REGIONS
+        Type                 : TABLE
+        Key field            : KSQL_INTERNAL_COL_0|+|KSQL_INTERNAL_COL_1
+        Key format           : STRING
+        Timestamp field      : Not set - using <ROWTIME>
+        Value format         : AVRO
+        Kafka topic          : PAGEVIEWS_REGIONS (partitions: 4, replication: 1)
+
+        Field    | Type
+        --------------------------------------
+        ROWTIME  | BIGINT           (system)
+        ROWKEY   | VARCHAR(STRING)  (system)
+        GENDER   | VARCHAR(STRING)
+        REGIONID | VARCHAR(STRING)
+        NUMUSERS | BIGINT
+        --------------------------------------
+
+        Queries that write into this TABLE
+        -----------------------------------
+        CTAS_PAGEVIEWS_REGIONS_3 : CREATE TABLE pageviews_regions         WITH (value_format='avro') AS       SELECT gender, regionid , COUNT(*) AS numusers       FROM pageviews_enriched         WINDOW TUMBLING (size 30 second)       GROUP BY gender, regionid       HAVING COUNT(*) > 1;
+
+        For query topology and execution plan please run: EXPLAIN <QueryId>
+
+        Local runtime statistics
+        ------------------------
+        messages-per-sec:      3.06   total-messages:      1827     last-message: 7/19/18 4:17:55 PM UTC
+        failed-messages:         0 failed-messages-per-sec:         0      last-failed:       n/a
+        (Statistics of the local KSQL server interaction with the Kafka topic PAGEVIEWS_REGIONS)
+        ksql>
+
+.. basics_tutorial_03_end 
+
+.. terminate_and_exit__start
+
 ------------------
 Terminate and Exit
 ------------------
 
 KSQL
 ----
 
-**Important:** Queries will continuously run as KSQL applications until
-they are manually terminated. Exiting KSQL does not terminate persistent
+**Important:** Persisted queries will continuously run as KSQL applications until
```

Comment:
Persistent?

Reply:
I actually prefer `persisted` here. /cc @JimGalasyn for comment